VulnerableApp_SAST_ExpectedIssues.csv

```diff
@@ -0,0 +1,30 @@
+CWE | Vulnerability Type | File | Line | Number of Sources
```

Comment:
@agigleux  what does number of sources mean ?

Reply:
Can we use https://docs.github.com/en/github/managing-files-in-a-repository/rendering-csv-and-tsv-data instead of | as a separator for better rendering ?

Reply:
"Number of Sources": I used that because in VulnerableApp, there are a couple of test cases where the "sink" is part of a method which is an utility method. So it means only one issue will be raised on this utility method whereas the attacks, the "sources", will come from multiple entry points (which are represented by each method annotated with @VulnerableAppRequestMapping). So if the SAST engine is raising one issue it's not enough to say all problems were detected. The SAST engine should also detect 5 entry points.
This is well visible in CommandInjection.java

Reply:
I updated the separator to comma to be compatible with https://docs.github.com/en/github/managing-files-in-a-repository/rendering-csv-and-tsv-data

Reply:
Yeah this number of sources problem is what i am also thinking from quite few days. Actually I think it is very complex to automate. please let me know if you have some idea about it.
Leaving that discussion aside, As per my understanding CWEs are not there for all the vulnerabilities so i think in future we might need to add more [classifiers](https://gist.github.com/bkimminich/5fc27e974ffa0add8bde279f45a4ae47#file-vulns-subtyped-flags-yml-L7).

@agigleux one more thing, can you please share report of scanner for VulnerableApp as i need to analyse on what other things can be a flag for vulnerabilities instead of line numbers.